scripts/teamDataScript.js

```diff
@@ -135,74 +135,76 @@ function loadPit()
 
+    $('#pitQuestionTableBody').html('');
     var localStorageObjName = pitAnswerObjName(season,event,currTeam);
     var team_QA_pairs = JSON.parse(localStorage.getItem(localStorageObjName)); //get QA pairs that may/may not have already been stored
-    var teamsAlreadyStoredBool = false;
-    if (checkForNull(team_QA_pairs))
+    
+    var alreadyStored = false;
+    var QA_keys = [];
+    if (checkForNull(team_QA_pairs)) //if data is stored
     {
-        teamsAlreadyStoredBool = true;
+        alreadyStored = true;
+        QA_keys = Object.keys(team_QA_pairs);
     }
 
-    for (var i = 0; i < questions.length;i++)
+    for (var i = 0; i < questions.length; i++) //iterate through questions array in cookie
     {
         var question = questions[i];
-        if (question.trim().length == 0) return;
-        
-        //loop through already stored local object: if question has an answer already stored then put that in the input box. Else, leave it empty (to fill in)
-        var potentialAnswer = '';
-        if (teamsAlreadyStoredBool == false) //len of loc storage obj == 0, nothing stored yet
+        var answer = ''; //default, in case something is stored
+        if (alreadyStored)
         {
-            potentialAnswer = '';
-        } else //len of loc storage obj = 1, something stored inside, now find answer to curr question
-        {
-            for (var j = 0; j < team_QA_pairs.length; j++)
+            if (QA_keys.includes(question)) //if answer is saved for it yet
             {
-                var pair = team_QA_pairs[j];
-                var answer = pair[question];
-                if (!checkForNull(answer))
-                {
-                    continue;
-                }
-                potentialAnswer = answer;
+                answer = team_QA_pairs[question];
             }
         }
 
-        var inputHTML = `<input type="text" value="${potentialAnswer}" class="answer">`; //pot answer is either '' or what is stored
-        var newTRHtml = `<tr><td class='question scoutingTableRow'>${question}</td><td class='scoutingTableRow'>${inputHTML}</td></tr>`;
+        var classText = 'w3-col l6 w6 s12 question w3-center'; 
+        var inputHTML = `<textarea width='50em' style='resize:vertical' value='${answer}' class='answer'>${answer}</textarea>`; //textarea
+        var insideHtml = `<div class='${classText}'><h4>${question}</h4></div> <div class='${classText}'>${inputHTML}</div>`; //both divs
         
-        $('#pitQuestionTable').append(newTRHtml);
+        var w3RowClassText = 'w3-row scoutingTableRow tableBorders';
+
+        //only put a top border on the first question row for styling: making sure top/bottom borders dont overlap for non-first rows
+        var newTRHtml = (i==0)?`<div class='${w3RowClassText}'> ${insideHtml} </div>`:
+                                `<div class='${w3RowClassText} tableBorders-NoTop'> ${insideHtml} </div>`;
```

Comment:
You can use css to give you this.

This line becomes:

```js
var newTRHtml = `<div class='${w3RowClassText} tableRowBorders-NoTop'> ${insideHtml} </div>`
```

And add to your css:
```css
table .tableRowBorders-NoTop:not(:first-child) {
  border-top:none;
}
```

Where, now you able to say for all elements in a table with the class `tableRowBorders-NoTop` - except the first one, apply this rule.